projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/kubernetes/ControlKubernetesService.java

```diff
@@ -5,8 +5,10 @@
 
 package com.vmware.taurus.service.kubernetes;
 
+import com.vmware.taurus.base.FeatureFlags;
 import com.vmware.taurus.service.KubernetesService;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
what is the problem with autowiring feature flags config?
btw, not sure if base module needs to be released first in case modified;

Reply:
I was getting errors due to the fact that autowiring requires an instance of the class, whereas I was using the autowired object inside the constructor. I went with the current approach as it seemed simpler (and it worked). I'm open to suggestions though.